lib/command.js

```diff
@@ -32,7 +32,7 @@ function checkValidInstall(name) {
         console.error(
             chalk.yellow('Ghost-CLI only works with Ghost versions >= 1.0.0.'),
             '\nIf you are trying to upgrade Ghost LTS to 1.0.0 ' +
-            `please see ${chalk.blue.underline('https://docs.ghost.org/v1/docs/migrating-to-1-0-0')}.`,
+            `please see ${chalk.blue.underline('https://docs.ghost.org/v0.11/docs/migrating-to-ghost-version-100')}.`,
```

Comment:
FWIW the 1.0 documentation has more in it - therefore think using the v1 migration doc is better. What needs to be fixed is the url - which is actually https://docs.ghost.org/v1/docs/migrating-to-ghost-1-0-0.